DSharpPlus/Enums/Message/MessageType.cs

```diff
@@ -123,6 +128,11 @@ public enum MessageType : int
         /// </summary>
         ApplicationCommand = 20,
 
+        /// <summary>
+        /// Message indicating a thread starter message.
+        /// </summary>
+        ThreadStarterMessage = 19,
```

Comment:
ThreadStarterMessage is 21; Reply is Type 19, as documented [on the docs](https://discord.com/developers/docs/resources/channel#message-object-message-types)

Reply:
oops, sry